modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/JSON.mustache

```diff
@@ -105,6 +109,10 @@ public class JSON {
             .registerTypeAdapter(OffsetDateTime.class, offsetDateTimeTypeAdapter)
             .registerTypeAdapter(LocalDate.class, localDateTypeAdapter)
             {{/jsr310}}
+            {{#java8}}
+            .disableHtmlEscaping(){{! Added to prevent base64 = (equal sign) from being escaped }}
```

Comment:
The gson default is to escape html (it is the contrary for Jackson) so this should be a choice left to the SDK user.